KIPs/kip-1.md

```diff
@@ -69,7 +69,7 @@ Following is the process that a successful KIP will move along:
 
 Each status change is requested by the KIP author and reviewed by the KIP editors. Use a pull request to update the status. Please include a link to where people should continue discussing your KIP. The KIP editors will process these requests as per the conditions below.
 
-* **Idea** -- Once the champion has asked the Klaytn community whether an idea has any chance of support, they will write a draft KIP as a [pull request](https://github.com/klaytn/KIPs/pulls). Consider including an implementation if this will aid people in studying the KIP.
+* **Idea** -- Once the champion has asked the Kaia community in  https://devforum.kaia.io/c/kips whether an idea has any chance of support, they will write a draft KIP as a [pull request](https://github.com/kaiachain/KIPs/pulls). Consider including an implementation if this will aid people in studying the KIP.
```

Comment:
Can you add `TODO` or `TBD` around here? Since the link will be changed.

Reply:
Added TBD for forum link